### PR TITLE
Make the app code checker a NOOP

### PR DESCRIPTION
**Pull request by ChristophWurst**
For 21 we decided to drop this outdated checker and it was even documented: https://github.com/nextcloud/documentation/pull/5505

so let's actually remove the functionality and "error out" every time it's called to make people aware. With Nextcloud 23 we can then fully drop it.

### CODE REVIEW

**Review by rullzer — APPROVED** [review]
:broom:

File: lib/private/App/CodeChecker/LanguageParseChecker.php [hunk 1]
```diff
@@ -1,60 +0,0 @@
...
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program. If not, see <http://www.gnu.org/licenses/>.
- *
- */
-
-namespace OC\App\CodeChecker;
-
-class LanguageParseChecker {
```
Comment (nickvergessen) [inline lib/private/App/CodeChecker/LanguageParseChecker.php:26]: I guess this thing here should be checked in some other way. But it's the only useful thing.

**Review by nickvergessen — APPROVED** [review]

File: lib/private/App/CodeChecker/LanguageParseChecker.php (hunk 1 above)
Comment (ChristophWurst) [reply lib/private/App/CodeChecker/LanguageParseChecker.php:26]: Was there ever a time where this didn't pass?

Comment (nickvergessen) [reply lib/private/App/CodeChecker/LanguageParseChecker.php:26]: yes, that's why it was written in first place. It happens when non-asci translators add half-utf8 chars which are not valid utf4 chars.

Comment (ChristophWurst) [reply lib/private/App/CodeChecker/LanguageParseChecker.php:26]: Okay, but that we can do in a simple php script and a github action

**Pull request merged**